docs/reference/filters.md

```diff
@@ -730,6 +730,27 @@ Example:
 * -> logHeader("request", "response") -> "https://www.example.org";
 ```
 
+### logBody
+
+The `logBody` filter logs the request or response body while
+streaming.  Note that this filter should be used only in diagnostics
+setup and with care, since the request and response body may contain
+sensitive data, and they also can explode the amount of logs. You can
+log request or response bodies. This filter has close to no overhead
+other than the I/O created by the logger.
+
+Parameters:
+
+* "request" or "response" (string varargs)
```

Comment:
Maybe we should support single argument for simplicity and let users chain `logBody("request") -> logBody("response")` if they need.
Also maybe we should add second limit parameter to cap number of bytes logged (and log all when it is zero).

Reply:
Do you see that we limit it? Why should chaining not work?

Reply:
We may also want to add default limit as a flag and spec option in the future.

Reply:
added a test

Reply:
Ok makes sense then I will restrict it to one argument.

Reply:
done